bean-validation/3.0/_index.md

```diff
@@ -1,25 +1,24 @@
 ---
-title: "Jakarta Bean Validation 3.0 (Milestone)"
-date: 2020-06-09
+title: "Jakarta Bean Validation 3.0"
+date: 2020-07-07
 summary: "Release for Jakarta EE 9"
 ---
 Jakarta Bean Validation defines a metadata model and API for JavaBean and method validation.
 
 * [Jakarta Bean Validation 3.0 Release Record](https://projects.eclipse.org/projects/ee4j.bean-validation/releases/3.0)
   * [Jakarta EE Platform 9 Release Plan](https://eclipse-ee4j.github.io/jakartaee-platform/jakartaee9/JakartaEE9ReleasePlan)
-* [Jakarta Bean Validation 3.0 Specification Document](./bean-validation-specification-3.0.0-M1.pdf) (PDF)
-* [Jakarta Bean Validation 3.0 Specification Document](./bean-validation-specification-3.0.0-M1.html) (HTML)
+* [Jakarta Bean Validation 3.0 Specification Document](./bean-validation-specification-3.0.0.pdf) (PDF)
+* [Jakarta Bean Validation 3.0 Specification Document](./bean-validation-specification-3.0.0.html) (HTML)
```

Comment:
According to our [Maven Versioning Rules](https://wiki.eclipse.org/JakartaEE_Maven_Versioning_Rules), the Spec should only be `<major>.<minor>` version, not three digits.  The APIs, TCKs, and CIs can be three digits, but the Spec should only be two digits.